test/unit/component/parser/json/BpmnJsonParser.sequenceFlow.conditional.test.ts

```diff
@@ -14,12 +14,11 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+import type { BpmnJsonModel } from '@lib/model/bpmn/json/BPMN20';
```

Comment:
**suggestion:** move this import close to the other imports related to _lib/model_ for consistency